tests/test_sota_checkpoints.py

```diff
@@ -90,7 +90,21 @@ def get_metric_file_name(model_name: str):
     @staticmethod
     def q_dq_config(config):
         nncf_config = NNCFConfig.from_json(config)
-        nncf_config["export_to_onnx_standard_ops"] = True
+        if "compression" in nncf_config:
+            compression_config = nncf_config["compression"]
+            quantization_config = None
```

Comment:
I have error TypeError: 'NoneType' with resnet18_imagenet_binarization_dorefa.json: It has "compression" but it not list, and "algorithm" is not "quantization", need more conditions.

Reply:
Done